pubnub-chat-impl/src/commonMain/kotlin/com/pubnub/chat/internal/MembershipImpl.kt

```diff
@@ -110,12 +118,14 @@ data class MembershipImpl(
             user,
             update.custom?.value ?: custom,
```

Comment:
actually I noticed an error in my code, it should be:
`update.custom.let { newCustom -> if (newCustom != null) newCustom.value else custom },`

and the new ones (status and type) should follow the same pattern